seatunnel-connectors-v2/connector-hbase/src/main/java/org/apache/seatunnel/connectors/seatunnel/hbase/format/HBaseDeserializationFormat.java

```diff
@@ -60,16 +60,15 @@ private Object deserializeValue(SeaTunnelDataType<?> typeInfo, byte[] cell) {
             case TINYINT:
             case SMALLINT:
             case INT:
-                return Bytes.toInt(cell);
+                return Integer.valueOf(Bytes.toString(cell));
             case BOOLEAN:
-                return Bytes.toBoolean(cell);
+                return Boolean.valueOf(Bytes.toString(cell));
```

Comment:
**Issue**: Similar to the integer conversion, converting bytes to string to boolean could cause performance issues and errors if the byte array does not represent a valid boolean string. <br> **Fix**: Use direct byte to boolean conversion to ensure efficiency and correctness. <br> **Code Suggestion**: 
 ```
 -                return Boolean.valueOf(Bytes.toString(cell));
 +                return Bytes.toBoolean(cell);
 ```